<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">@import url(https://themes.googleusercontent.com/fonts/css?kit=FskwCFJX4ojWovwWYLwsTg);.lst-kix_1hbtbmuc8o2r-8>li:before{content:"\0025a0   "}.lst-kix_1hbtbmuc8o2r-6>li:before{content:"\0025cf   "}ul.lst-kix_1hbtbmuc8o2r-8{list-style-type:none}.lst-kix_1hbtbmuc8o2r-4>li:before{content:"\0025cb   "}.lst-kix_1hbtbmuc8o2r-5>li:before{content:"\0025a0   "}.lst-kix_1hbtbmuc8o2r-3>li:before{content:"\0025cf   "}li.li-bullet-0:before{margin-left:-18pt;white-space:nowrap;display:inline-block;min-width:18pt}.lst-kix_1hbtbmuc8o2r-0>li:before{content:"\0025cf   "}.lst-kix_1hbtbmuc8o2r-1>li:before{content:"\0025cb   "}.lst-kix_1hbtbmuc8o2r-2>li:before{content:"\0025a0   "}ul.lst-kix_1hbtbmuc8o2r-7{list-style-type:none}ul.lst-kix_1hbtbmuc8o2r-6{list-style-type:none}ul.lst-kix_1hbtbmuc8o2r-5{list-style-type:none}ul.lst-kix_1hbtbmuc8o2r-4{list-style-type:none}ul.lst-kix_1hbtbmuc8o2r-3{list-style-type:none}ul.lst-kix_1hbtbmuc8o2r-2{list-style-type:none}.lst-kix_1hbtbmuc8o2r-7>li:before{content:"\0025cb   "}ul.lst-kix_1hbtbmuc8o2r-1{list-style-type:none}ul.lst-kix_1hbtbmuc8o2r-0{list-style-type:none}ol{margin:0;padding:0}table td,table th{padding:0}.c20{border-right-style:solid;padding-top:0pt;border-top-width:0pt;border-right-width:0pt;padding-left:0pt;padding-bottom:0pt;line-height:1.2;border-left-width:0pt;border-top-style:solid;border-left-style:solid;border-bottom-width:0pt;border-bottom-style:solid;orphans:2;widows:2;text-align:left;padding-right:0pt;height:16pt}.c36{border-right-style:solid;padding-top:0pt;border-top-width:0pt;border-right-width:0pt;padding-left:0pt;padding-bottom:0pt;line-height:1.2;border-left-width:0pt;border-top-style:solid;border-left-style:solid;border-bottom-width:0pt;border-bottom-style:solid;text-align:left;padding-right:0pt}.c22{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:bottom;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;background-color:#ffffff;border-left-style:solid;border-bottom-width:1pt;width:63.8pt;border-top-color:#000000;border-bottom-style:solid}.c18{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:bottom;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;background-color:#ffffff;border-left-style:solid;border-bottom-width:1pt;width:62.2pt;border-top-color:#000000;border-bottom-style:solid}.c4{border-right-style:solid;padding:2pt 2pt 2pt 2pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:bottom;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;background-color:#ffffff;border-left-style:solid;border-bottom-width:1pt;width:76.5pt;border-top-color:#000000;border-bottom-style:solid}.c19{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:62.2pt;border-top-color:#000000;border-bottom-style:solid}.c32{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:76.5pt;border-top-color:#000000;border-bottom-style:solid}.c9{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:63.8pt;border-top-color:#000000;border-bottom-style:solid}.c5{color:#000000;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Arial";font-style:normal}.c0{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Arial";font-style:normal}.c26{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:13pt;font-family:"Arial";font-style:normal}.c39{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:15pt;font-family:"Arial";font-style:normal}.c33{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:20pt;font-family:"Arial";font-style:normal}.c1{padding-top:18pt;padding-bottom:6pt;line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.c2{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:16pt;font-family:"Arial";font-style:normal}.c3{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:10pt;font-family:"Arial";font-style:normal}.c16{color:#434343;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:14pt;font-family:"Arial";font-style:normal}.c6{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c17{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:justify}.c12{padding-top:12pt;padding-bottom:12pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c28{padding-top:16pt;padding-bottom:4pt;line-height:1.15;page-break-after:avoid;text-align:justify}.c38{padding-top:20pt;padding-bottom:6pt;line-height:1.15;page-break-after:avoid;text-align:justify}.c23{padding-top:16pt;padding-bottom:4pt;line-height:1.15;page-break-after:avoid;text-align:left}.c37{padding-top:20pt;padding-bottom:6pt;line-height:1.15;page-break-after:avoid;text-align:left}.c34{padding-top:18pt;padding-bottom:6pt;line-height:1.15;page-break-after:avoid;text-align:justify}.c13{padding-top:0pt;padding-bottom:0pt;line-height:1.0;text-align:left}.c11{padding-top:0pt;padding-bottom:0pt;line-height:1.15;text-align:center}.c25{text-decoration-skip-ink:none;-webkit-text-decoration-skip:none;color:#1155cc;text-decoration:underline}.c8{border-spacing:0;border-collapse:collapse;margin-right:auto}.c31{font-size:9pt;font-weight:400;font-family:"DM Sans"}.c40{background-color:#ffffff;max-width:451.4pt;padding:72pt 72pt 72pt 72pt}.c27{margin-left:36pt;padding-left:0pt}.c24{color:inherit;text-decoration:inherit}.c30{padding:0;margin:0}.c35{width:33%;height:1px}.c21{orphans:2;widows:2}.c15{font-size:10pt;font-weight:700}.c14{color:#ffffff}.c29{height:14pt}.c7{height:0pt}.c10{height:11pt}.title{padding-top:0pt;color:#000000;font-size:26pt;padding-bottom:3pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.subtitle{padding-top:0pt;color:#666666;font-size:15pt;padding-bottom:16pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:20pt;color:#000000;font-size:20pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h2{padding-top:18pt;color:#000000;font-size:16pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h3{padding-top:16pt;color:#434343;font-size:14pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h4{padding-top:14pt;color:#666666;font-size:12pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h5{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h6{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;font-style:italic;orphans:2;widows:2;text-align:left}</style></head><body class="c40 doc-content"><h1 class="c21 c37" id="h.6kq6ak3gjd4d"><span class="c33">Benchmarking Results for Shape-Based Similarity Analysis on Complexes</span></h1><p class="c6 c10"><span class="c0"></span></p><p class="c6"><span class="c0">Grisell D&iacute;az Leines &amp; Sameer Velankar</span></p><p class="c6 c10"><span class="c0"></span></p><p class="c17"><span class="c31">Protein Data Bank in Europe, European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL- EBI), Wellcome Genome Campus, Hinxton, Cambridge CB10 1SD, United Kingdom </span></p><h2 class="c20" id="h.cqcnsbhpdtu1"><span class="c2"></span></h2><h2 class="c21 c36" id="h.mwdgg2h2ronp"><span class="c2">SARS-CoV-2 Spike complex </span></h2><p class="c6 c10"><span class="c0"></span></p><h3 class="c21 c28" id="h.8s478rcp3q2"><span class="c16">Benchmarking set </span></h3><p class="c6"><span>The benchmarking set for the SARS-CoV-2 post-fusion S2 Spike complex (PDB-CPX-140202) focuses on the trimeric spike glycoprotein of severe acute respiratory syndrome coronavirus 2. The full PDBe complex entry (</span><span class="c25"><a class="c24" href="https://www.google.com/url?q=https://www.ebi.ac.uk/pdbe/pdbe-kb/complexes/PDB-CPX-140202?utm_source%3Dchatgpt.com&amp;sa=D&amp;source=editors&amp;ust=1769434290276692&amp;usg=AOvVaw23_dOX7TZj0QOr-dlvUvCq">https://www.ebi.ac.uk/pdbe/pdbe-kb/complexes/PDB-CPX-140202</a></span><span class="c0">) includes over 400 structural assemblies, representing a wide range of conformational states and binding contexts. For this preliminary analysis, we filtered assemblies to retain only those with titles containing &ldquo;RBD&rdquo; and a resolution of 3 &Aring; or better. This selection is intended to capture high-quality structures that reflect the biologically relevant open (RBD-Up) and closed (RBD-Down) conformations of the spike complex. The resulting benchmark set consists of approximately 50 assemblies, providing a manageable yet representative dataset for early-stage comparison and evaluation.</span></p><h3 class="c23 c21" id="h.qjqn7xojmawg"><span class="c16">Clustering</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 558.58px; height: 339.00px;"><img alt="" src="images/image1.png" style="width: 558.58px; height: 353.52px; margin-left: 0.00px; margin-top: -14.52px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></h3><p class="c6 c10"><span class="c0"></span></p><p class="c6"><span class="c15">Figure 1.</span><span class="c3">&nbsp;Hierarchical clustering of SARS-CoV-2 Spike protein conformations.</span></p><p class="c6"><span class="c3">(Left) Global and local similarity scores were computed between pairs of entries using 3D Zernike and spectral descriptors. Scores range from 0 (highly similar) to 1 (highly dissimilar).</span></p><p class="c6"><span class="c3">(Right) Dendrogram with representative molecular surfaces illustrating the two major conformational clusters: 3-RBD-Down (red) and 3-RBD-Up (blue).</span></p><p class="c6 c10"><span class="c0"></span></p><p class="c17 c10"><span class="c0"></span></p><p class="c17"><span class="c0">The two main clusters were successfully identified using a similarity score defined as a combination of global (Zernike) and local (spectral) descriptors. There are two false positives for 3-RBD-Up &nbsp;(9by0 and 9cxe) identified with the global/local similarity score. </span></p><h3 class="c28 c21" id="h.bu5nkwx1g1du"><span class="c16">Local similarity for different conformations </span></h3><p class="c6 c10"><span class="c0"></span></p><p class="c6"><span class="c0">Spectral descriptors enable the detection of local structural similarity between different conformations of the Spike protein. Despite large global rearrangements, these descriptors identify conserved local regions correspondences. The figure illustrates a point-to-point correspondence between two distinct conformations, revealing partial structural similarity despite flexibility.</span></p><p class="c6 c10"><span class="c0"></span></p><p class="c6 c10"><span class="c0"></span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 601.70px; height: 338.67px;"><img alt="" src="images/image4.png" style="width: 601.70px; height: 338.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c15">Figure 2.</span><span class="c3">&nbsp;Functional map and local point-to-point correspondence between the 3RBD-up and 3RBD-down conformations of the SARS-CoV-2 spike complex.</span></p><p class="c6"><span class="c3">(Left) Functional map (correspondence matrix) between PDB entries 7V7F (3RBD-up) and 7V7D (3RBD-down). The strong diagonal signal indicates regions of high local similarity despite the conformational transition. (Right) Point-to-point correspondence between the molecular surfaces of 7V7F and 7V7D. For clarity, only 80 correspondence lines are shown. Circles highlight regions where local similarity is preserved across conformations, illustrating that spectral descriptors can detect partial structural correspondence even under large conformational changes.</span></p><p class="c6 c10"><span class="c3"></span></p><p class="c6 c10"><span class="c3"></span></p><p class="c6 c10"><span class="c3"></span></p><h2 class="c21 c34" id="h.rd2ngaxmtd47"><span class="c2">Bacterial ribosome </span></h2><h3 class="c21 c23" id="h.5e803spcjblh"><span class="c16">Benchmarking Set </span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 333.74px; height: 354.78px;"><img alt="" src="images/image6.png" style="width: 333.74px; height: 354.78px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></h3><p class="c6"><span class="c0">The bacterial ribosome from Thermus thermophilus is included as a benchmarking case study to evaluate the performance of our method on large ribonucleoprotein (RNP) complexes. The ribosome forms a ~50-subunit oligomeric assembly, representing a substantially more complex and heterogeneous system than protein-only complexes. The benchmarking dataset is based on a manually curated collection of ribosome structures spanning distinct functional states. These structures capture four primary functional states, classical pre-translocation, hybrid, chimeric hybrid, and classical post-translocation, which are characterized by different tRNA configurations and pronounced large-scale conformational changes, including intersubunit rotation, small subunit head rotation, and L1 stalk movement. This dataset provides a stringent test of the ability of shape-based and spectral methods to identify global and local similarity in the presence of extensive flexibility and compositional complexity.</span></p><p class="c6 c10"><span class="c0"></span></p><p class="c6"><span class="c0">The annotated functional states in the benchmarking dataset are shown in Table 1.</span></p><p class="c6 c10"><span class="c0"></span></p><table class="c8"><tr class="c7"><td class="c32" colspan="1" rowspan="1"><p class="c13"><span class="c5">Classical</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c13"><span class="c5">Hybrid </span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c13"><span class="c5">Chimeric hybrid</span></p></td></tr><tr class="c7"><td class="c4" colspan="1" rowspan="1"><p class="c11"><span class="c3">4V51_1</span></p></td><td class="c18" colspan="1" rowspan="1"><p class="c11"><span class="c3">4V8O_1</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c11"><span class="c3">4V9K_1</span></p></td></tr><tr class="c7"><td class="c4" colspan="1" rowspan="1"><p class="c11"><span class="c3">4V6F_2</span></p></td><td class="c18" colspan="1" rowspan="1"><p class="c11"><span class="c3">4V90_1</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c11"><span class="c3">4V9L_1</span></p></td></tr><tr class="c7"><td class="c4" colspan="1" rowspan="1"><p class="c11"><span class="c3">4V7L_2</span></p></td><td class="c18" colspan="1" rowspan="1"><p class="c11"><span class="c3">4V9H_1</span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c11"><span class="c3">4V9M_1</span></p></td></tr><tr class="c7"><td class="c4" colspan="1" rowspan="1"><p class="c11"><span class="c3">4V8U_1</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c13 c10"><span class="c0"></span></p></td><td class="c22" colspan="1" rowspan="1"><p class="c11"><span class="c3">4V9J_1</span></p></td></tr><tr class="c7"><td class="c4" colspan="1" rowspan="1"><p class="c11"><span class="c3">4V63_1</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c10 c13"><span class="c0"></span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c13 c10"><span class="c0"></span></p></td></tr><tr class="c7"><td class="c4" colspan="1" rowspan="1"><p class="c11"><span class="c3">4V5F_1</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c13 c10"><span class="c0"></span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c13 c10"><span class="c0"></span></p></td></tr><tr class="c7"><td class="c4" colspan="1" rowspan="1"><p class="c11"><span class="c3">4V67_2</span></p></td><td class="c19" colspan="1" rowspan="1"><p class="c13 c10"><span class="c0"></span></p></td><td class="c9" colspan="1" rowspan="1"><p class="c13 c10"><span class="c0"></span></p></td></tr></table><p class="c6 c10"><span class="c0"></span></p><p class="c6"><span class="c0">Table 1. PDB entries and assembly identifiers corresponding to the annotated functional states in the benchmarking dataset.</span></p><h3 class="c23 c21 c29" id="h.b27stpxemgs3"><span class="c16"></span></h3><h3 class="c23 c21" id="h.kmvbfbrqd6r"><span class="c16">Clustering</span></h3><h2 class="c1" id="h.79vzj3w52qi2"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 601.70px; height: 349.33px;"><img alt="" src="images/image3.png" style="width: 601.70px; height: 349.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></h2><p class="c6"><span class="c15">Figure 4</span><span class="c3">. Hierarchical clustering of Bacterial ribosome conformations. (Left) Global and local similarity scores were computed between pairs of entries using 3D Zernike and spectral descriptors. Scores range from 0 (highly similar) to 1 (highly dissimilar).(Right) Dendrogram with representative molecular surfaces (colored by entity identifier) illustrating 4 major conformational clusters: Classical Pre- and post-translocation, Chimeric and Hybrid groups. </span></p><p class="c6 c10"><span class="c0"></span></p><p class="c6"><span class="c0">We compute a similarity score that integrates global shape information (3D Zernike descriptors) with local geometric features (spectral descriptors). Hierarchical clustering using the Ward linkage in Fig. 2 reveals a clear separation between chimeric hybrid and classical pre- and post-translocation conformations based on both global and local features. In contrast, the classical post-translocation state exhibits local similarity to hybrid structures and therefore clusters appear in proximity to the hybrid conformations in the dendrogram.</span></p><p class="c6"><span class="c0">A cutoff was chosen to allow a maximum of four clusters, however, defining an a priori threshold to clearly distinguish between classical post-translocation and hybrid state remains challenging due to their shared local structural features.</span></p><p class="c6"><span class="c5">Clustering analysis using OPTICS algorithm:</span></p><p class="c6 c10"><span class="c5"></span></p><p class="c6 c10"><span class="c5"></span></p><p class="c6"><span>To further validate the clustering obtained from surface descriptors, we applied the OPTICS algorithm, which does not require a predefined cutoff. This analysis consistently identified five clusters and an isolated structure 4V8O, supporting the robustness of the observed conformational clustering using surface descriptors.</span></p><p class="c6 c10"><span class="c0"></span></p><p class="c6"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 411.00px; height: 345.81px;"><img alt="" src="images/image5.png" style="width: 411.00px; height: 371.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span class="c15">Figure 5. </span><span class="c3">OPTICS clustering of the score matrix shown in a 2D MDS embedding. Noise points are shown in gray. Clusters reflect the density structure in the original distance space. The embedding is for visualization only.</span></p><p class="c6 c10"><span class="c0"></span></p><p class="c6"><span class="c0">List of clusters obtained with OPTICS:</span></p><p class="c6 c10"><span class="c0"></span></p><p class="c6"><span class="c0">Cluster 0 : 4V51, 4V7L (Classical pre-translocation)</span></p><p class="c6"><span class="c0">Cluster 1 : 4V63, 4V67, 4V6F (Classical pre-translocation)</span></p><p class="c6"><span class="c0">Cluster 2: 4V5F,4V8U (Classical post-translocation )</span></p><p class="c6"><span class="c0">Cluster 3: 4V90,4V9H (Hybrid)</span></p><p class="c6"><span class="c0">Cluster 4: 4V9J, 4V9K, 4V9L, 4V9M (Chimeric)</span></p><p class="c6"><span class="c0">Cluster X: 4V8O &nbsp;(Hybrid, considered noise as it cannot be classified in a cluster of at least 2 elements)</span></p><p class="c6 c10"><span class="c0"></span></p><p class="c6 c10"><span class="c26"></span></p><p class="c6 c10"><span class="c3"></span></p><h2 class="c1" id="h.b1mibnq1h126"><span class="c2">Hemoglobin HbA complex</span></h2><p class="c6 c10"><span class="c39"></span></p><p class="c6"><span class="c0">The benchmarking set for human hemoglobin focuses on its tetrameric assembly (&alpha;&#8322;&beta;&#8322;) derived from Homo sapiens. The PDBe complex entry (PDB-CPX-154652) includes over 300 structural assemblies, representing various functional and intermediate states. Hemoglobin is known to undergo cooperative conformational changes between two primary states&mdash;the relaxed (R) state and the tense (T) state, as well as several intermediate conformations.</span></p><p class="c6"><span class="c0">To obtain a manageable and functionally relevant subset for benchmarking, assemblies were filtered using the following criteria:</span></p><ul class="c30 lst-kix_1hbtbmuc8o2r-0 start"><li class="c6 c27 li-bullet-0"><span class="c0">Journal annotations containing the substring &ldquo;state T&rdquo; or &ldquo;state R&rdquo;</span></li><li class="c6 c27 li-bullet-0"><span class="c0">Resolution &le; 3 &Aring;</span></li></ul><p class="c6"><span class="c0">This filtering approach is expected to yield approximately 30 assemblies, providing a representative dataset for assessing local and global similarity across functional states.</span></p><p class="c6"><span>Reference: PDBe complex entry: </span><span class="c25"><a class="c24" href="https://www.google.com/url?q=https://www.ebi.ac.uk/pdbe/pdbe-kb/complexes/PDB-CPX-154652&amp;sa=D&amp;source=editors&amp;ust=1769434290289905&amp;usg=AOvVaw0XNMFPPG7q40rTPdUJUxuS">https://www.ebi.ac.uk/pdbe/pdbe-kb/complexes/PDB-CPX-154652</a></span></p><h3 class="c23 c21" id="h.m3udq71fkpuf"><span>Clustering</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 601.70px; height: 400.00px;"><img alt="" src="images/image2.png" style="width: 601.70px; height: 400.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></h3><p class="c6"><span class="c15">Figure 6.</span><span class="c3">&nbsp;Hierarchical clustering of protein conformations for Hemoglobin HbA complex.</span></p><p class="c6"><span class="c3">(Left) Global and local similarity scores computed between pairs of entries using 3D Zernike and spectral descriptors. Scores range from 0 (highly similar) to 1 (highly dissimilar).</span></p><p class="c6"><span class="c3">(Right) Dendrogram with representative molecular surfaces illustrating the two major conformational clusters: T-state (red) and R-State (blue).</span></p><p class="c6 c10"><span class="c0"></span></p><p class="c6"><span class="c0">We successfully distinguish the T-state and R-state conformations using a combination of spectral and 3D Zernike descriptors. The resulting similarity scores show no false-positive similarities between structures annotated as different functional states within the benchmarking dataset. </span></p><p class="c6 c10"><span class="c3"></span></p><p class="c6 c10"><span class="c0"></span></p><p class="c6 c10"><span class="c0"></span></p><p class="c6 c10"><span class="c0"></span></p><p class="c6 c10"><span class="c0"></span></p><h1 class="c21 c38" id="h.vwwlzbln2ubx"><span class="c33">Protein interfaces</span></h1><p class="c12"><span class="c0">Ubiquitin serves as an illustrative example of a protein chain that participates in multiple, distinct protein&ndash;protein complexes while maintaining a conserved interaction surface. In the three PDB entries considered here (2JY6, 2DEN), ubiquitin adopts a highly similar global fold and engages binding partners through the canonical hydrophobic interaction patch centered on Ile44.</span></p><p class="c12"><span class="c0">In PDB entries 2JY6 and 2DEN, ubiquitin binds to UBA (ubiquitin-associated) domains from different proteins. Although these UBA domains originate from distinct proteins, they share a common fold and recognize ubiquitin in a similar binding mode, resulting in comparable local interface geometries. These interactions exemplify conserved ubiquitin recognition by structurally related binding domains.</span></p><p class="c12"><span class="c14">ross structurally diverse proteins.</span><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 601.70px; height: 409.33px;"><img alt="" src="images/image7.png" style="width: 601.70px; height: 409.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c6"><span>Fig. 1 | Local similarity between two ubiquitin-binding protein partners. (Top) Protein complexes showing ubiquitin bound to chain A in complex 2DEN and to chain B in complex 2JY6. (Bottom) Local shape correspondence between the ubiquitin-binding partners (chains 2DEN-A and 2JY6-B), highlighting a high degree of partial structural similarity in the interface regions that engage ubiquitin. </span><span class="c3">For clarity, &nbsp;80 correspondence lines are shown.</span></p><p class="c6 c10"><span class="c3"></span></p><p class="c12 c10"><span class="c0"></span></p><p class="c6"><span class="c0">Local spectral descriptors enable the identification of &nbsp;interaction patterns between ubiquitin-binding proteins even when their global folds differ. In the example shown, ubiquitin interacts with two partner proteins (2DEN-A and 2JY6-B) &nbsp;that have distinct overall structures, yet the regions contacting ubiquitin exhibit high local geometric similarity. By focusing on local surface patches rather than full-chain alignments, the descriptors capture shared shape and interface features in the ubiquitin-binding regions that are not apparent from global structural comparisons. This demonstrates how local spectral descriptors can reveal interface-level and binding-site similarity across structurally different proteins, highlighting similarities that are functionally relevant but globally masked.</span></p><p class="c6 c10"><span class="c0"></span></p><p class="c6 c10"><span class="c0"></span></p><p class="c6 c10"><span class="c0"></span></p><p class="c6 c10"><span class="c0"></span></p><p class="c6"><sup><a href="#ftnt1" id="ftnt_ref1">[1]</a></sup></p><div><p class="c6 c10"><span class="c0"></span></p></div><hr class="c35"><div><p class="c13 c21"><a href="#ftnt_ref1" id="ftnt1">[1]</a><span class="c3">&nbsp;These results are preliminary and are not intended for redistribution. For questions or reuse requests, please get in touch with Grisell D&iacute;az Leines (gdiazleines@ebi.ac.uk) and Sameer Velankar (sameer@ebi.ac.uk).</span></p></div></body></html>